data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,57 @@ mission "Arachi Cartoonist 2"
+			choice
+				`	"I've always got time for a story, lead the way."`
+				`	"I'm not much of a fan of plays, I'll pass."`
+					decline
+			`	You follow him outside the spaceport city, and deep into a nearby forest. On the way there, several other Saryds in ancient garbs are guiding others who were also intrigued by the offer.`
+			`	You arrive at a clearing with three different paths to take, and you can make out a faint light deep within each of them. After a few minutes it seems as if all the Saryds involved in the celebration have arrived. They grab three caskets from behind a bush, and lay them before you and the others they brought here.`
+			`	"Now friends, choose your path, you must," one of the Saryds says, and many from the crowd grab small objects from the caskets, seemingly already familiar with this. You move forward to look into the caskets, and see that each one has different things inside: the left one is full of seeds, the middle one full of seashells, and the right one full of feathers.`
+			choice
+				`	(Grab a seed.)`
+					goto seed
+				`	(Grab a seashell.)`
+					goto pearl
+				`	(Grab a feather.)`
+			`	Once you have the feather in your hands, the organizers tell you to go through the path on the right, and you do so.`
+			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it. You keep your eyes on the wooden stage in front of you, about a third of the size of human opera stages. Several lights shine onto it, and as you look up you can only barely see some spotlights high up in the trees. After the audience is all here, the curtains open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below, or strings from above. There are figures close to where the curtains were, and others further into the stage, the various layers granting depth to the scenery. After a brief moment, the show begins.`
+			`	Entering a clearing, a Saryd huntress with gray hair stalks a large bird resting on a rock. She moves on the borders of the clearing, hiding behind bushes and small trees. She looks for the best spot from which to snipe at the bird, her bow and arrow ready in her hands. It's only when she is nearing a large entrance to the clearing, on the opposite side of where she entered, that she finds the spot, and takes aim.`
+			`	She is disrupted when she hears a loud roar, and jumps to the ground to narrowly avoiding the beast that had come from behind her. You're unsure how accurate the paper's depiction of the creature is, but you can only describe it as a hedgelion: a bit larger than lions from Earth, with various quills where its mane should be. The quills continue in a line on its back, reaching the tip of the tail.`
```

Comment:
```suggestion
			`	She is disrupted when she hears a loud roar, and jumps to the ground, narrowly avoiding the beast that had come from behind her. You're unsure how accurate the paper's depiction of the creature is, but you can only describe it as a hedgelion: a bit larger than lions from Earth, with various quills where its mane should be. The quills continue in a line on its back, reaching the tip of the tail.`
```

Oh hell. How did I make this mistake? I'm embarassed. Good thing I looked over this again.